modules/@ergonode/comments/src/components/Forms/CommentsForm.vue

```diff
@@ -131,7 +138,7 @@ export default {
         },
         isAllowedToUpdate() {
```

Comment:
what about migrating from it to specific business? And just passing prop disabled